resin/resin/imgui/sdf_tree.cpp

```diff
@@ -116,6 +121,36 @@ void SDFTreeComponentVisitor::visit_group(::resin::GroupNode& node) {
     is_any_node_clicked_ = true;
   }
 
+  if (ImGui::IsItemClicked(ImGuiMouseButton_Right)) {
+    ImGui::OpenPopup("GroupPopUpMenu");
+  }
+
+  if (ImGui::BeginPopup("GroupPopUpMenu")) {
+    if (ImGui::Selectable("Save as prefab")) {
+      auto curr_id                                                                   = node.node_id();
+      auto name                                                                      = node.name();
+      auto& sdf_tree                                                                 = sdf_tree_;
+      std::vector<std::pair<const std::string_view, const std::string_view>> filters = {
```

Comment:
I'd separate this line from the previous three assignments to improve readability

Reply:
Additionally, does this filter have to be created here? Shouldn't it be a constant std::array?

Reply:
> Additionally, does this filter have to be created here? Shouldn't it be a constant std::array?

To be exact: constant as in static const in class